internal/logging/logger.go

```diff
@@ -0,0 +1,72 @@
+package logging
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/rs/zerolog"
+)
+
+var Logger zerolog.Logger
+
+func init() {
+	nop := zerolog.Nop()
+	SetGlobalLogger(nop)
```

Comment:
why not `SetGlobalLogger(zerolog.Nop())`?

Reply:
leftover from it being a pointer. Instafix!